### DISCUSSION TITLE
How to config vite?

### DISCUSSION THREAD

**Discussion by PeterlitsZo** [storybookjs/storybook#17500] [Help]
In this issue: https://github.com/system-ui/theme-ui/issues/2131, I find that `storybook` do not read `vite.config.js`, would you please tell how to config storybook's vite?

I try to read your document but cannot find anything useful about config...

**Comment by khuezy** [comment]
Have you tried this: https://github.com/eirslett/storybook-builder-vite/issues/85#issuecomment-900831050

**Reply by PeterlitsZo** [reply]
Thanks for your swift reply, I will try it next weekend.
